src/Symfony/Component/Serializer/Tests/Fixtures/DummyObjectWithEnumConstructor.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace Symfony\Component\Serializer\Tests\Fixtures;
+
+use Symfony\Component\Serializer\Tests\Fixtures\StringBackedEnumDummy;
+
+class DummyObjectWithEnumConstructor
+{
+    public function __construct(public StringBackedEnumDummy $get) 
+    {
+    }
+}
```

Comment:
missing newline at EOF (this can probably be fixed globally by configuring your favorite editor)